test/data/test_builtin_datasets.py

```diff
@@ -157,6 +157,108 @@ def test_multi30k(self):
                                 "multi30k_task*.tar.gz")
         conditional_remove(datafile)
 
+    def test_udpos_sequence_tagging(self):
+        from torchtext.experimental.datasets import UDPOS
+
+        # smoke test to ensure imdb works properly
+        train_dataset, valid_dataset, test_dataset = UDPOS()
+        self.assertEqual(len(train_dataset), 12543)
+        self.assertEqual(len(valid_dataset), 2002)
+        self.assertEqual(len(test_dataset), 2077)
+        assert_allclose(train_dataset[0][0][:10],
```

Comment:
Just to bring your attention. We have a PR to support `self.assertEqual` for torch.tensor. https://github.com/pytorch/text/pull/822

Reply:
noted, will wait that PR merged firs then

Reply:
You could add [TODO] tag in the comment line and move forward with this PR.